CliFx/Domain/HelpTextWriter.cs

```diff
@@ -246,21 +246,45 @@ void RenderOptions()
 
                     RenderColumnIndent();
 
-                    // Description
-                    if (!string.IsNullOrWhiteSpace(option.Description))
-                    {
-                        Render(option.Description);
-                        Render(" ");
-                    }
+                    RenderOptionDescription(option);                    
+
+                    RenderNewLine();
+                }
+            }
+
+            void RenderOptionDescription(CommandOptionSchema option)
+            {
+                // Description
+                if (!string.IsNullOrWhiteSpace(option.Description))
+                {
+                    Render(option.Description);
 
-                    // Environment variable
-                    if (!string.IsNullOrWhiteSpace(option.EnvironmentVariableName))
+                    // Valid values for enum types.
+                    // Property can actually be null here due to damn it operator in CommandArgumentSchema.cs:103,106.
+                    var propertyType = option.Property?.PropertyType;
+
+                    if (propertyType is object && propertyType.IsEnum)
                     {
-                        Render($"(Environment variable: {option.EnvironmentVariableName}).");
-                        Render(" ");
+                        // Cast to object so that we can use linq methods on the Array.
+                        var values = Enum.GetValues(propertyType)
+                            .Cast<object>()
+                            .Select(v => v.ToString().ToLowerInvariant());
```

Comment:
I think it's better not to do `ToLowerInvariant()` in case the enum fields consist of more than one word (e.g. `Validation.ThrowOnNonZeroExitCode` would turn into `throwonnonzeroexitcode` which is just unreadable).

Reply:
Moved to `CommandArgumentSchema` and removed call to `ToLowerInvariant()`.